matcher.go

```diff
@@ -366,6 +366,7 @@ func (m *Matcher) Match(data []byte) (bool, error) {
 		return false, err
 	}
 
+	//	fmt.Printf("Parse node: %v\n", m.def.ParseNode)
```

Comment:
This is not part of the actual code, I don't think its intended to be included.

Reply:
Removed.